### ISSUE TITLE
Folder.jpg not working for cover art

### ISSUE THREAD

**Issue by Hk1020** [Rello/audioplayer#256]
### Expected behavior
Albums should show cover art contained in file Folder.jpg as per the documentation

### Actual behavior
No cover art is shown

### Steps to reproduce the behavior
1. Have a folder with mp3 files which don't include cover art and a file Folder.jpg
2. scan for music
3. show albums

### Additional information (optional)
If Folder.jpg is renamed to folder.jpg the cover art is shown after reset library/scan for audio files. The same behavior exists for cover.jpg

### Server configuration
**Operating system**:
linux

**Web server:**
apache

**Database:**
mysql

**PHP version:**

**Nextcloud/ownCloud version:** (see `/status.php`)
12.0.3

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Audio Player version:** (see `/settings/apps`)
2.2.1

**Where did you install Audio Player from:**

**Are you using external storage, if yes which one:** `local/smb/sftp/...`

**Are you using encryption:** yes/no

### Client configuration
**Operating system:**

**Browser:**

#### Nextcloud/ownCloud log
<details>
<summary>Nextcloud/ownCloud log (`/data/[nextcloud|owncloud].log`)</summary>

```json
Insert your Nextcloud/ownCloud log here
```

</details>

#### Web server error log (optional)
<details>
<summary>Web server error log</summary>

```
Insert your web server log here
```

</details>

#### Browser log (optional)
<details>
<summary>Browser log</summary>

```
Insert your browser log here
```

</details>

**Comment by ghost** [comment]
@Hk1020 In the [documentation](https://github.com/rello/audioplayer/wiki/how-are-id3-tags-processed#cover-art) is nothing of file names with capital letters. It works as described!

**Comment by Hk1020** [comment]
@xraMsamohT You are right, I didn't read it carefully enough. But Windows Media Player writes Folder.jpg in every directory, so no artwork shown.

**Comment by Hk1020** [comment]
I just checked the latest commits. Why don't you simply make all file names case insensitive? Any reason to distinguish Folder.jpg and folder.jpg? If you deploy on Windows it'll work that way anyway.

**Comment by Rello** [comment]
hello,
we are dependent in NC standard classes to not scan the filecache table (or even the files locally) every time. and this class is case dependent to derive a unique file path for every file.
because one folder can have a `Folder.jpg` and a `folder.jpg` and they might have a different meaning

**Comment by Rello** [comment]
Released with 2.2.2
